editor/effects.js

```diff
@@ -40,15 +41,11 @@ export default {
 		} );
 	},
 	REQUEST_POST_UPDATE_SUCCESS( action ) {
-		const { post } = action;
-		const [ baseUrl, query ] = window.location.href.split( '?' );
-		const qs = parse( query || '' );
-		if ( qs.post_id === post.id ) {
+		const { post, isNew } = action;
+		if ( ! isNew ) {
 			return;
 		}
-
-		const newUrl = baseUrl + '?' + stringify( {
-			...qs,
+		const newUrl = getGutenbergURL( {
```

Comment:
Inconsistency between `Url` and `URL`. I think we'll want to settle on and use consistently a recommendation for acronyms. The core guidelines do not include one. Calypso recommends strict camel-casing, but I've personally grown fonder of upper-case. I can create an issue.